herddb-utils/src/test/java/herddb/utils/SQLRecordPredicateFunctionsTest.java

```diff
@@ -41,6 +41,13 @@ public void testCompareAndLike() throws Exception {
         assertTrue(SQLRecordPredicateFunctions.like("test", "%es%"));
         assertFalse(SQLRecordPredicateFunctions.like("tesst", "te_t"));
         assertTrue(SQLRecordPredicateFunctions.like("test", "te_t"));
+        assertTrue(SQLRecordPredicateFunctions.like("bar (foo)", "%(foo)%"));
```

Comment:
please also add a test without any '%' and '_'

it may sound weird but it is quite common in generated SQL